modules/installation-and-upgrade/pages/container-deployment/suma/migrate-suma-to-a-container.adoc

```diff
@@ -11,20 +11,20 @@ It is not possible to perform an in-place migration.
 
 The original server is referred to as the **source server,** while the newly set-up machine is designated as the **destination server.**
 
-The migration procedure currently does not include any hostname renaming functionality. Consequently, the fully qualified domain name (FQDN) on the new server will remain identical to that on the source server. Therefore, following migration, it will be necessary to manually adjust the DNS records to point to the new server.
+The migration procedure currently does not include any hostname renaming functionality. Consequently, the fully qualified domain name (FQDN) on the destination server will remain identical to that on the source server. Therefore, following migration, it will be necessary to manually adjust the DNS records to point to the new server.
```

Comment:
```suggestion
The migration procedure currently does not include any hostname renaming functionality.
The fully qualified domain name (FQDN) of the destination server will remain identical to the one of the source server.
Therefore, after the migration, it will be necessary to manually adjust the DNS records to point to the new server.
```